be/src/runtime/data_stream_recvr.h

```diff
@@ -166,6 +167,7 @@ class DataStreamRecvr {
 
     // vectorized::SortedChunksMerger merges chunks from different senders.
     std::unique_ptr<vectorized::SortedChunksMerger> _chunks_merger;
+    std::unique_ptr<vectorized::CascadeChunkMerger> _cascade_merger;
```

Comment:
why need _cascade_merger and _chunks_merger at the same time?

Reply:
No the same time. `chunks_merger` is for non-pipeline mode, and `cascade_merger` is for pipeline-mode.

Why not replace `_chunks_merger` in non-pipeline mode? Cause it works in blocking manner, but pipeline mode works in non-blocking manner, that would need different implementations.

Reply:
I see, would better comment it.